GetMessage/handler.ts

```diff
@@ -252,19 +303,33 @@ export function GetMessageHandler(
     }
     const maybeMessageStatus = errorOrMaybeMessageStatus.right;
 
-    const returnedMessage = {
-      message,
-      notification: pipe(notificationStatuses, O.toUndefined),
-      // we do not return the status date-time
-      status: pipe(
-        maybeMessageStatus,
-        O.map(messageStatus => messageStatus.status),
-        // when the message has been received but a MessageStatus
-        // does not exist yet, the message is considered to be
-        // in the ACCEPTED state (not yet stored in the inbox)
-        O.getOrElse(() => MessageStatusValueEnum.ACCEPTED)
-      )
-    };
+    const returnedMessage = pipe(
+      {
+        message,
+        notification: pipe(notificationStatuses, O.toUndefined),
+        // we do not return the status date-time
+        status: pipe(
+          maybeMessageStatus,
+          O.map(messageStatus => messageStatus.status),
+          // when the message has been received but a MessageStatus
+          // does not exist yet, the message is considered to be
+          // in the ACCEPTED state (not yet stored in the inbox)
+          O.getOrElse(() => MessageStatusValueEnum.ACCEPTED)
+        )
+      },
+      // Enrich message info with advanced properties if user is allowed to read them
+      messageWithoutAdvacedProperties =>
```

Comment:
typo in the variable name, propagated in lines 323 and 325

Reply:
Nice catch! Done in [20fae5a](https://github.com/pagopa/io-functions-services/pull/202/commits/20fae5a64590785abc8512c76fda4f681bbab510)